core/store/migrate/migrations/0090_ocr2_relay.sql

```diff
@@ -0,0 +1,28 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE offchainreporting2_oracle_specs
+    ADD COLUMN relay text,
+    ADD COLUMN relay_config JSONB,
+    DROP COLUMN evm_chain_id,
+    DROP CONSTRAINT chk_contract_address_length;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN contract_address TO contract_id;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN transmitter_address TO transmitter_id;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN encrypted_ocr_key_bundle_id TO ocr_key_bundle_id;
```

Comment:
Another bytea => text candidate?

Reply:
no this is always bytea, all ocr2 bundles are still sha256 hashes of key material